### DISCUSSION TITLE
Issue while running auto generate migration on mysql

### DISCUSSION THREAD

**Discussion by exasky** [mmRoshani/sequelize-typescript-migration#16] [General]
When passing that line 4 of 'getLastMigrationState.js' while running practice.ts, I have the following error
```
code: 'ER_PARSE_ERROR',
errno: 1064,
sqlState: '42000',
sqlMessage: `You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '"SequelizeMeta" ORDER BY name desc limit 1' at line 1`,
sql: 'SELECT name FROM "SequelizeMeta" ORDER BY name desc limit 1',
```

If I remove double quotes around SequelizeMeta, everything works as expected

**Comment by ChocolateVS** [comment]
Yes i have the same issue and did the same thing, with mariadb also. I might make a branch and create npm pacakage so that i dont have to keep making that fix...
